README.md

```diff
@@ -178,7 +178,7 @@ public enum StudentGroup {
 
 The project structure should look like this:
 
-![project structure](https://curriculum-content.s3.amazonaws.com/6036/java-mod-5-jpa/studentgroup_enum.png)
+![project structure](https://curriculum-content.s3.amazonaws.com/6036/java-mod-5-jpa/studentgroup.png)
```

Comment:
Maybe crop the image a little more unless you want the file path to be kind of visible next to the `jpa-example` folder

Reply:
ok, updated.